### ISSUE TITLE
[Content styleguide] Content doesn't align with style guidance in VA.gov content style guide. (05.08.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#28328]
#### VFS team name
Healthcare Experience

#### VFS product name
Check-in MVP

#### Point of Contact/Reviewers
Peggy Gannon

---

### Platform Issue
Content doesn't align with style guidance in VA.gov content style guide.

### Issue Details
there shouldn't be punctuation at the end H1 and section heads

### Supporting Guidance
[05.08.2](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
Please remove the period from this H1: Check in with a staff member

### VFS Team Tasks to Complete

- [x] Comment on the ticket if there are questions or concerns
- [x] VFS team closes the ticket when the issue has been resolved

**Comment by jparker0425** [comment]
Will address post MVP launch.

**Comment by mdewey** [comment]
This should be resolve in staging.

**Comment by shiragoodman** [comment]
adding labels so ticket is consistent with post-pilot process

**Comment by jparker0425** [comment]
Issue resolved.
